src/cms/views/users/user_view.py

```diff
@@ -101,22 +102,40 @@ def post(self, request, *args, **kwargs):
                 or user_form.cleaned_data["is_staff"]
                 or user_profile_form.cleaned_data["regions"]
             ):
-                user = user_form.save()
-                user_profile_form.save(user=user)
-
-                if user_form.has_changed() or user_profile_form.has_changed():
-                    if user_instance:
-                        messages.success(request, _("User was successfully saved"))
+                # check if either activation link is send or password is set or user already exists
+                if (
+                    user_profile_form.cleaned_data["send_activation_link"]
+                    or (
+                        not user_profile_form.cleaned_data["send_activation_link"]
+                        and user_form.cleaned_data["password"]
+                    )
```

Comment:
```suggestion
                    or user_form.cleaned_data["password"]
```
If an if-block has multiple `OR` conditions, Python will go into the if-block as soon as it sees a condition which evaluates to `True`. This means that on every subsequent condition, you can be sure that the first condition evaluates to `False` and you don't have to test for it...